dqc_us_rules/dqc_us_0006.py

```diff
@@ -37,30 +47,55 @@ def validate_dates_within_periods(val):
                                 ruleVersion=_RULE_VERSION)
 
 
-def _date_range_check(check_types, check_dei, date_bounds_dict, modelXbrl):
+def _date_range_check(check_types, check_dei, date_bounds_dict, model_xbrl):
     """
-    Takes two lists of fact names, a dict of date boundaries and modelXbrl and then compiles a list of all
-    facts in the modelXbrl that match the names in the supplied name lists. It then compares the context
-    date span to the date boundaries for the corresponding document period focus. Any facts with spans less than
-    or larger than the supplied boundaries are returned in a dict based on the document period focus.
+    Takes two lists of fact names, a dict of date boundaries and modelXbrl and
+    then compiles a list of all facts in the modelXbrl that match the names in
+    the supplied name lists. It then compares the context date span to the date
+    boundaries for the corresponding document period focus. Any facts with
+    spans less than or larger than the supplied boundaries are returned in a
+    dict based on the document period focus.
+
+    :param check_types: first list of names
+    :type check_types: list of str
+    :param check_dei: second list of names
+    :type check_dei: list of str
+    :param date_bounds_dict: period from which names should be pulled from
+    :type date_bounds_dict: dictionary of date bounds
+    :param model_xbrl: model xbrl from which facts are to be pulled from
+    :type model_xbrl: ModelXbrl
+    :rtype: dict of facts
+    :return: list of facts that match the names in the two lists and are within
+    the date_bounds_dict
     """
     facts_in_error = defaultdict(list)
-    list_of_facts = facts.LegalEntityAxis_facts_by_member(facts.get_facts_with_type(check_types, modelXbrl))
-    list_of_facts = _dict_list_update(list_of_facts, (facts.LegalEntityAxis_facts_by_member(facts.get_facts_dei(check_dei, modelXbrl))))
+    list_of_facts = facts.LegalEntityAxis_facts_by_member(
+        facts.get_facts_with_type(check_types, model_xbrl)
+    )
+    list_of_facts = _dict_list_update(
+        list_of_facts,
+        (facts.LegalEntityAxis_facts_by_member(facts.get_facts_dei(
+            check_dei,
+            model_xbrl)
+        ))
+    )
 
-    dfpf_list = facts.lookup_dei_facts('DocumentFiscalPeriodFocus', modelXbrl)
+    dfpf_list = facts.lookup_dei_facts('DocumentFiscalPeriodFocus', model_xbrl)
     dfpf_dict = facts.LegalEntityAxis_facts_by_member(dfpf_list)
     for lea_member, fact_list in list_of_facts.items():
-        lookup = lea_member if lea_member in dfpf_dict else facts.LEGALENTITYAXIS_DEFAULT
+        lookup = lea_member \
```

Comment:
instead of using the \ to break up the line, should put parens around the line and break it up like normal